spec.bs

```diff
@@ -763,70 +763,258 @@ enrollment and authentication. This extends the
 
-    * In the absence of such as UX, a browser can require a
-        user gesture as part of initiating the API.
+NOTE: In addition the above, the assertion `challenge` will also be incorrect.
 
-# Privacy Considerations # {#sctn-privacy-considerations}
+The general concept here is that a Secure Payment Confirmation assertion is
+essentially useless unless it is part of an ongoing online transaction.
 
-Note: This section is still very much a draft.
+## Merchant-supplied authentication data ## {#sctn-security-merchant-data}
 
-SPC primarily points to WebAuthn for its privacy model, but it does relax
-that model in significant ways that should be considered.
+A consequence of this specification's third-party authentication ceremony is
+that even in a valid transaction (i.e. one that the [=Relying Party=] is
+expecting), a third-party provides the transaction details that are shown to
+the user:
 
-* Probing for credential ids
+* Transaction amount and currency
+* Payment instrument name and icon
+* Payee origin
 
-    * Currently SPC exits immediately (on Chrome) when no passed credential
-        matches the current device. This could be used to probe for whether the
-        current device matches an existing known credential. This does show a
-        very clear payments UX when a match is found (though that may be too late).
+This could lead to a spoofing attack, in which a merchant presents incorrect
+data to the user. For example, the merchant could tell the bank (in the
+backend) that it is initiating a purchase of $100, but then pass $1 to the SPC
+API (and thus show the user a $1 transaction to verify).
 
-        Note: The instant-exit may be removed in the future.
+Secure Payment Confirmation actually makes defeating this kind of attack easier
+than it currently is on the web. In online payments today, the bank has to
+trust that the merchant showed the user the correct amount in their checkout
+flow (and any fraud discoveries are post-payment, when the user checks their
+account statement).
 
-* Identifying user across different payment instruments.
+With Secure Payment Confirmation, the bank can (and should) instead
+[[#sctn-verifying-assertion|verify the cryptogram]] that they receive, to
+ensure that it aligns with the transaction details that the merchant initiated
+the transaction for.
 
-     * If the relying party uses the same credentials for a given user across
-         multiple payment instruments, this could expose that linkage to the
-         merchant (as they could track what sets of credentials they see for
-         each instrument they ask about on the backend). See https://github.com/w3c/secure-payment-confirmation/issues/77
+# Privacy Considerations # {#sctn-privacy-considerations}
+
+As this specification builds on top of WebAuthn, the
+[[webauthn-3#sctn-privacy-considerations|WebAuthn Privacy Considerations]]
+are applicable to this specification. The below subsections comprise the
```

Comment:
Suggest deleting "to this specification" (to match earlier phrasing and because it's shorter).